packages/kmod-5.15-nvidia/nvidia-fabricmanager.cfg

```diff
@@ -0,0 +1,34 @@
+# Modern, systemd-aware settings:
```

Comment:
I wonder if users will ask us to provide an API to allow changes to this configuration file. I would recommend take a note to keep this in mind.

Reply:
Good call out. We can definitely revisit if any customer requires api to change some of the settings. It would be part of a separate PR.